### PR TITLE
Add configuration for probot-stale

### PR DESCRIPTION
**Pull request by jasonrudolph**
Using https://github.com/atom/atom/pull/14382 as inspiration, this PR configures the [probot-stale integration](https://github.com/probot/stale) to automatically close issues that have not had any recent activity.

### CODE REVIEW

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  are closed. If you would like this issue to remain open:
+
+   1. Verify that you can still reproduce the issue using the latest version of this package with the latest version of Atom
+   1. Comment that the issue is still reproducible and include:
+      * What version of the package you reproduced the issue on
+      * What version of Atom you reproduced the issue on
+      * What OS and version you reproduced the issue on
+      * What steps you followed to reproduce the issue
+
+  Issues that are labeled as triaged will not be automatically marked as stale.
```
Comment (Ben3eeE) [inline .github/stale.yml]: This does not match up with the `exemptLabels`. Why did you remove `regression` and `triaged` from `exemptLabels`?

Comment (jasonrudolph) [reply .github/stale.yml]: @Ben3eeE: Thanks for pointing this out. https://github.com/atom/sort-lines/pull/89 should get us back on track.
